woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -1861,6 +1861,22 @@ def test_accessor_drop(sample_df):
     assert single_input_df.ww.schema == list_input_df.ww.schema
     assert to_pandas(single_input_df).equals(to_pandas(list_input_df))
 
+    inplace_df = schema_df.copy()
```

Comment:
This test is getting pretty big. I think it would be better to test the `inplace` behavior in a separate test. Separating these will help us in the future so we will more clearly see what is being evaluated by each test.

Reply:
done!